sdk/keyvault/test-resources.json

```diff
@@ -206,53 +206,53 @@
             "objectId": "[parameters('testApplicationOid')]",
             "permissions": {
               "keys": [
-                "backup",
-                "create",
-                "decrypt",
-                "delete",
-                "encrypt",
-                "get",
-                "getrotationpolicy",
-                "import",
-                "list",
-                "purge",
-                "recover",
-                "restore",
-                "rotate",
-                "sign",
-                "setrotationpolicy",
-                "unwrapKey",
-                "update",
-                "verify",
-                "wrapKey"
+                "Get",
```

Comment:
We shouldn't be modifying content in keyvault. Or this should be in another PR.

Reply:
Rollbacked unnecessary fixes